File: MinecraftPvpPlugin/src/main/java/me/minecraft/minecraftpvpplugin/Items.java
package me.minecraft.minecraftpvpplugin;

import org.bukkit.Material;
import org.bukkit.entity.Item;
import org.bukkit.inventory.ItemStack;
import org.bukkit.inventory.meta.ItemMeta;

public class Items {
    public static ItemStack IronSword = new ItemStack(Material.IRON_SWORD);
    public static ItemStack DiamondSword = new ItemStack(Material.DIAMOND_SWORD);

    public static ItemStack DiamondPickaxe = new ItemStack(Material.DIAMOND_AXE);
    public static ItemMeta SwordItemMeta = Items.DiamondSword.getItemMeta();
    public static ItemStack IronHelmet = new ItemStack(Material.IRON_HELMET);
    public static ItemStack IronChestplate = new ItemStack(Material.IRON_CHESTPLATE);
    public static ItemStack IronLeggings = new ItemStack(Material.IRON_LEGGINGS);
    public static ItemStack IronBoots = new ItemStack(Material.IRON_BOOTS);
    public static ItemStack FishingRod = new ItemStack(Material.FISHING_ROD);
    public static ItemStack Gapple = new ItemStack(Material.GOLDEN_APPLE, 5);
}

File: MinecraftPvpPlugin/src/main/java/me/minecraft/minecraftpvpplugin/Items.kt
package me.minecraft.minecraftpvpplugin

import org.bukkit.Material
import org.bukkit.inventory.ItemStack
import org.bukkit.inventory.meta.ItemMeta

object Items {
    var ironSword: ItemStack = ItemStack(Material.IRON_SWORD)
    var diamondSword: ItemStack = ItemStack(Material.DIAMOND_SWORD)

    var diamondPickaxe: ItemStack = ItemStack(Material.DIAMOND_AXE)
    var swordItemMeta: ItemMeta = diamondSword.itemMeta
    var ironHelmet: ItemStack = ItemStack(Material.IRON_HELMET)
    var ironChestplate: ItemStack = ItemStack(Material.IRON_CHESTPLATE)
    var ironLeggings: ItemStack = ItemStack(Material.IRON_LEGGINGS)
    var ironBoots: ItemStack = ItemStack(Material.IRON_BOOTS)
    var fishingRod: ItemStack = ItemStack(Material.FISHING_ROD)
    var gapple: ItemStack = ItemStack(Material.GOLDEN_APPLE, 5)
}
